### PR TITLE
The car stops at red traffic lights (but it's very slow due to the "Camera" being turned ON in the simulation)

### LINKED ISSUE #2: Add simplest TL classifier (from walkthrough)

### PR DESCRIPTION
**Pull request by MTDzi**
Closes #2

### CODE REVIEW

File: ros/src/tl_detector/tl_detector.py [hunk 1]
```diff
@@ -131,23 +144,36 @@ def process_traffic_lights(self):
...
-        #TODO find the closest visible traffic light (if one exists)
+        if self.pose:
+            position = self.pose.pose.position
+            car_wp_idx = self.get_closest_waypoint(position.x, position.y)
+            # TODO find the closest visible traffic light (if one exists)
+            diff = len(self.waypoints.waypoints)
+            for i, light in enumerate(self.lights):
+                # Get sto line waypoint index
+                line = stop_line_positions[i]
+                temp_wp_idx = self.get_closest_waypoint(line[0], line[1])
```
Comment (MTDzi) [inline ros/src/tl_detector/tl_detector.py:161]: @ank700 this is where the closest traffic light is found.
Or have you found an error?

Comment (ank700) [reply ros/src/tl_detector/tl_detector.py:161]: I found this section. Some parts of the tl_detector.py were missing in my fork and I got confused. I have cloned the whole code this time from the MD/simple_tl_controller branch.

Comment (MTDzi) [reply ros/src/tl_detector/tl_detector.py:161]: OK.

Can I accept this PR?

Comment (MTDzi) [reply ros/src/tl_detector/tl_detector.py:161]: Merging.

**Pull request merged**